packages/shared/src/components/modals/RanksModal/DevCardFooter.tsx

```diff
@@ -17,7 +17,7 @@ export default function DevCardFooter({
 }: DevCardFooterProps): ReactElement {
   return (
     <div className="pt-2">
-      <Modal.Subtitle>Generate your DevCard</Modal.Subtitle>
+      <Modal.Subtitle className="mb-2">Generate your DevCard</Modal.Subtitle>
```

Comment:
Yes, in the component where we declared this, we removed these margins. This is because it should be defined on the consuming end. Unless it is a padding, it should be from what was expelained.